### PR TITLE
WRN-5320: Add unit tests to MediaOverlay

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added unit tests for MediaOverlay

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-5320

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: MediaOverlay/tests/MediaOverlay-specs.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+		const mediaOverlayCaption = screen.getByText('This is a caption');
+		const captionContainer = screen.getByText('This is a caption').parentElement.parentElement;
+
+		expect(mediaOverlayCaption).toBeInTheDocument();
+		expect(captionContainer).toHaveClass('caption');
+	});
+
+	test('should render a text', () => {
+		const text = 'The quick brown fox jumped over the lazy dog. The bean bird flies at sundown.';
+		render(<MediaOverlay text={text} source="abc.mp4" />);
```
Comment (stanca-pop-lgp) [inline MediaOverlay/tests/MediaOverlay-specs.js:29]: Please sort props.

File: MediaOverlay/tests/MediaOverlay-specs.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+		const expected = 'flex-start';
+		const actual = screen.getByText(text).parentElement.parentElement.parentElement;
+
+		expect(actual).toHaveStyle({'align-items': expected});
+	});
+
+	test('should render a subtitle', () => {
+		const subtitle = '07:00 AM - 08:00 AM';
+		render(<MediaOverlay subtitle={subtitle} source="abc.mp4" />);
```
Comment (stanca-pop-lgp) [inline MediaOverlay/tests/MediaOverlay-specs.js:69]: Please sort props.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]
Minor changes required. Otherwise great job!

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**